lib/utils.js

```diff
@@ -168,7 +168,7 @@ module.exports.can_use_canvas = function can_use_canvas(createCanvas) {
 // TODO: remove after it's fixed in chrome for at least 2 releases
 module.exports.cib_can_use_region = function cib_can_use_region() {
   return new Promise(resolve => {
-    if (typeof createImageBitmap === 'undefined') {
+    if ((typeof Image === 'undefined') || (typeof createImageBitmap === 'undefined')) {
```

Comment:
Please add comment (about service worker), why this condition is mandatory. Without comments/tests such things are difficult to maintain.

```
// `Image` check required for use in `ServiceWorker`
```